guides/micronaut-cloud-oidc-base/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+  "publish": false,
+  "authors": ["Burt Beckwith"],
+  "tags": ["cloud", "security", "oauth2", "oidc"],
```

Comment:
probably `openid-connect` as a tag as well. not sure if people identify `oidc` as an acronym.